services/web/server/src/simcore_service_webserver/garbage_collector/_tasks_api_keys.py

```diff
@@ -4,20 +4,20 @@
 """
 import asyncio
 import logging
-from typing import AsyncIterator, Callable
+from collections.abc import AsyncIterator, Callable
 
 from aiohttp import web
 from tenacity import retry
 from tenacity.before_sleep import before_sleep_log
 from tenacity.wait import wait_exponential
 
-from .login.api_keys_db import prune_expired_api_keys
+from ..login.api_keys_db import prune_expired_api_keys
 
 logger = logging.getLogger(__name__)
 
 CleanupContextFunc = Callable[[web.Application], AsyncIterator[None]]
 
-_SEC = 1
+_SEC = 1  # in seconds
```

Comment:
Is this variable really useful? actually I see where this is used...
it looks a lot like something to make a linter shut up. would it not make more sense to have
```python
_MIN_WAITING_TIME_S = 5
_MAX_WAITING_TIME_S = 30
```